src/app/qgsdxfexportdialog.cpp

```diff
@@ -149,7 +149,7 @@ QgsVectorLayerAndAttributeModel::QgsVectorLayerAndAttributeModel( QgsLayerTree *
     const QgsVectorLayer *vLayer = qobject_cast< const QgsVectorLayer *>( QgsProject::instance()->mapLayer( id ) );
     if ( vLayer )
     {
-      mCreateDDBlockInfo[vLayer] = QgsDxfExportDialog::settingsDxfEnableDDBlocks->value();
+      mCreateDDBlockInfo[vLayer] = true;
```

Comment:
Do we still need this variable if it's always true (?) or is there still a way to disable?

Reply:
See https://github.com/qgis/QGIS/pull/56984#issuecomment-2031888845 makes sense now